Jenkins: Please set --httpsKeyStore

Using revision 9a79e50c49e9f961a39e7b2875b211210bbffa37.

```
Running from: /usr/share/jenkins/jenkins.war
webroot: EnvVars.masterEnvVars.get("JENKINS_HOME")
2022-11-22 11:43:22.308+0000 [id=1]	INFO	winstone.Logger#logInternal: Beginning extraction from war file
2022-11-22 11:43:22.438+0000 [id=1]	INFO	winstone.Logger#logInternal: Jetty shutdown successfully
java.io.IOException: Failed to start a listener: winstone.HttpsConnectorFactory
	at winstone.Launcher.spawnListener(Launcher.java:260)
	at winstone.Launcher.<init>(Launcher.java:188)
	at winstone.Launcher.main(Launcher.java:403)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at executable.Main._main(Main.java:334)
	at executable.Main.main(Main.java:116)
Caused by: winstone.WinstoneException: Please set --httpsKeyStore
	at winstone.AbstractSecuredConnectorFactory.configureSsl(AbstractSecuredConnectorFactory.java:54)
	at winstone.HttpsConnectorFactory.start(HttpsConnectorFactory.java:56)
	at winstone.Launcher.spawnListener(Launcher.java:254)
	... 8 more
2022-11-22 11:43:22.439+0000 [id=1]	SEVERE	winstone.Logger#logInternal: Container startup failed
winstone.WinstoneException: Please set --httpsKeyStore
	at winstone.AbstractSecuredConnectorFactory.configureSsl(AbstractSecuredConnectorFactory.java:54)
	at winstone.HttpsConnectorFactory.start(HttpsConnectorFactory.java:56)
	at winstone.Launcher.spawnListener(Launcher.java:254)
Caused: java.io.IOException: Failed to start a listener: winstone.HttpsConnectorFactory
	at winstone.Launcher.spawnListener(Launcher.java:260)
	at winstone.Launcher.<init>(Launcher.java:188)
	at winstone.Launcher.main(Launcher.java:403)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at executable.Main._main(Main.java:334)
	at executable.Main.main(Main.java:116)
jenkins4marcis exited with code 1
```

Using `jenkins/jenkins:lts` in `docker-compose.yml` could bring some unexpected results when the LTS version changes. It would be better to pin it to a specific LTS release.

>  Stable (LTS)
>
> Long-Term Support (LTS) release baselines are chosen every 12 weeks from the stream of regular releases

-- https://www.jenkins.io/download/

Automatic major version upgrades every 12 weeks are risky in a production environment.

Possible fixes:
- Use `jenkins/jenkins:2.164-jdk11`
- Set `httpsKeyStore` and pin Jenkins to the latest LTS version

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -152,8 +152,7 @@ services:
     restart: on-failure
 
   jenkins:
-    image: jenkins/jenkins:lts
-    # image: istresearch/jenkins:latest
+    image: jenkins/jenkins:2.164-jdk11
     container_name: jenkins4${COMPOSE_PROJECT_NAME}
     user: jenkins
     ports:

